src/routes/auth.route.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express'
+import { forgotPassword, resetPassword } from '../controllers/auth.controller'
+import { AuthSchema, validateSchema } from '../validations/auth.validation'
+const router = Router()
```

Comment:
I know that your asserting it below but let's do it here. Since we are using typescript we should have an index file in every folder to export files
```suggestion
const authRouter = Router();
```